schema/course.schema.json

```diff
@@ -0,0 +1,39 @@
+{
+  "$anchor": "vanilla-course",
```

Comment:
This was originally graphic-course. I presume it was copied from the graphic component. I changed this to be aligned with the rest of the schema files.